neutrinordp/xrdp-neutrinordp.h

```diff
@@ -25,6 +25,7 @@
 #include "xrdp_rail.h"
 #include "xrdp_client_info.h"
 #include "xrdp_constants.h"
+#include "ms-rdpbcgr.h"
```

Comment:
I don't think this include is necessary here.

Reply:
Certainly.
There were still some things listed during the trial and error process.
This line will be deleted.